api/v1alpha1/clusterextension_types.go

```diff
@@ -145,6 +145,8 @@ type ClusterExtensionStatus struct {
 	InstalledBundle *BundleMetadata `json:"installedBundle,omitempty"`
 	// +optional
 	ResolvedBundle *BundleMetadata `json:"resolvedBundle,omitempty"`
+	// +optional
+	ContentURL string `json:"contentURL,omitempty"`
```

Comment:
Is there some reason we need `ContentURL` in the cluster extension? The original purpose of this in the rukpak `Bundle` API was for BD provisioners to pull bundles from other bundle provisioners and operate on them.

Now that everything is being handled directly in `ClusterExtension` and there are no v1.0.0 requirements around exposing this, let's leave this out to keep our API exposure to the absolute minimum necessary.

Reply:
It was just to keep the API surface similar to BD. 

> The original purpose of this in the rukpak Bundle API was for BD provisioners to pull bundles from other bundle provisioners and operate on them.

Wasn't aware of this. I'll remove it and fix the linter